File: level1/예산 12982.py
def solution(d, budget):
    cnt = 0
    d.sort() # 신청금액 오름차순으로 정렬
    for i in d:
        if budget >= i: # 남은 예산보다 해당 부서의 신청금액이 작다면
            budget -= i # 신청금액을 지원한다.
            cnt += 1
    return cnt

File: level1/javascript/예산 12982.js
function solution(d, budget) {
    let cnt = 0;
    d.sort((a,b) => a-b);
    for(let i = 0; i < d.length; i++){
        if(d[i] <= budget){
            budget -= d[i];
            cnt++;
        } 
    }
    return cnt;
}

// 테스트 코드
console.log(solution([1,3,2,5,4], 9));